app/dashboard/models.py

```diff
@@ -252,6 +254,30 @@ def fetch_issue_description(self):
         if body:
             self.issue_description = body
 
+    def fetch_issue_comments(self, save=True):
```

Comment:
uses the `app.github` method to pull comments via API vs parsing html

Reply:
This has changed in:  https://github.com/gitcoinco/web/pull/156